test/fixtures/BootstrapCommand/extra/lerna.json

```diff
@@ -0,0 +1,8 @@
+{
+  "lerna": "__TEST_VERSION__",
+  "version": "1.0.0",
+  "packages": [
+    {"glob": "packages/*/package.json"},
+    {"glob": "package-3/package.json"}
```

Comment:
I would think that glob would be the name of a package here, but it looks like it globs paths to find a package.json